### PR TITLE
Enable funlen linter (part 4)

### PR DESCRIPTION
**Pull request by brojeg**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: [934](https://github.com/ydb-platform/ydb-go-sdk/issues/934)

## What is the new behavior?

-
-
-

### CODE REVIEW

File: internal/cmd/gtrace/writer.go
Comment (asmyasnikov) [inline internal/cmd/gtrace/writer.go:1]: I think tool gtrace no need to change because it is an ad-hoc tool

Comment (asmyasnikov) [reply internal/cmd/gtrace/writer.go:1]: And the breaking action was broken with your PR - because you change a code of gtrace tool

Comment (brojeg) [reply internal/cmd/gtrace/writer.go:1]: thank you for the review. I've rolled back [writer.go] and added //nolint:funlen where needed.

File: internal/credentials/static.go [hunk 1]
```diff
@@ -87,9 +87,7 @@ func (c *Static) Token(ctx context.Context) (token string, err error) {
 			fmt.Errorf("dial failed: %w", err),
 		)
 	}
-	defer func() {
-		_ = cc.Close()
-	}()
+	defer cc.Close()
```
Comment (asmyasnikov) [inline internal/credentials/static.go:90]: this broke errcheck linter check

**Review by asmyasnikov — APPROVED** [review]

**Pull request merged**